tests/unit/amo/components/TestCategory.js

```diff
@@ -272,14 +276,15 @@ describe(__filename, () => {
       },
     });
 
+    sinon.assert.calledWith(fakeDispatch, setViewContext(fakeCategory.type));
```

Comment:
Similarly, this test exercises [only a specific if branch](https://github.com/bobsilverberg/addons-frontend/blob/d41b5e99ba24219eab50f1a1127f937497195fa6/src/amo/components/Category/index.js#L113). It's not necessary to make an assertion about the view context logic since that's not affected by the if branch.